publishable/database/migrations/2017_11_26_013050_add_user_role_relationship.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddUserRoleRelationship extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->unsignedInteger('role_id')->nullable(false)->change();
```

Comment:
Do we set a default value anywhere? It's possible that someone could have users currently with a null value for `role_id`, which would cause this migration to break.

Reply:
I see, I had that doubt too. We should skip this, then?

Reply:
Yes, I would remove the change to remove nullability on this field.

Reply:
fixed